private/detectchannelnan.m

```diff
@@ -0,0 +1,35 @@
+function [badchannels] = detectchannelnan(cfg,data)
```

Comment:
if this is a private function, it should be called by multiple high-level functions. Otherwise it does not make sense to make a separate function out of it.

Reply:
oh, it is now (in this PR) called by two functions. So that is fine

Reply:
the code actually simplifies to a one-liner. No need to make a private function out of it.

Reply:
Depends on how elaborate the function should be. My version of detectchannelnan()
- only runs if there are any NaNs detected, which (notably) speeds up things for complete data.
- throws a warning for each channel that is classified as bad, alerting users
- combines the detect channels containing NaNs with any bad channels pre-specified by the user (which might not contain NaNs).

A one-liner without this functionality would of course be possible.
Is there anything more I should/ can do here?